tensorflow/compiler/xla/service/gpu/target_util.cc

```diff
@@ -166,7 +225,16 @@ llvm::CallInst* EmitCallToTargetIntrinsic(
   if (target_triple.isNVPTX()) {
     llvm_intrinsic_id = gpu_intrinsic_id.nvptx_intrinsic;
   } else if (target_triple.getArch() == llvm::Triple::amdgcn) {
-    llvm_intrinsic_id = gpu_intrinsic_id.amdgpu_intrinsic;
+    llvm::Intrinsic::ID* llvm_intrinsic_id_ptr;
+    if ((llvm_intrinsic_id_ptr = absl::get_if<llvm::Intrinsic::ID>(
```

Comment:
Please extract the assignment onto a separate line. This is consistent with the rest of the codebase and reads better, at least in my opinion.

Reply:
will do.